stable/fluent-bit/values.yaml

```diff
@@ -181,7 +181,16 @@ input:
   tail:
     memBufLimit: 5MB
     path: /var/log/containers/*.log
-
+  systemd:
+    enabled: true
```

Comment:
this is needed to be true to install the chart?

Reply:
No, I am presuming that everybody wants node logs, which may not be the case.
If set to false, no systemd input will be added thus no node logs will be configured.
Would you prefer it set to false by default?

Reply:
I think we can leave true. But did you test the chart setting that to false?

Reply:
No, let me test now. brb

Reply:
works as expected from what I see! no `systemd` input, and no mounted hostpath vol `/etc/machine-id`

Reply:
nice, thanks!